landing-page/vue.config.js

```diff
@@ -17,6 +17,12 @@ module.exports = {
                 changeOrigin: true,
                 secure: false,
                 logLevel: "debug"
+            },
+            '^/api/public/profiles': {
+                target: "http://server.cityvizor.cesko.digital:3000",
```

Comment:
Tohle je adresa konkretni pro aws prostredi a neni z ENV ?

Reply:
Tady je to asi v pohodě, to je nastavení proxy dev serveru, tím se zařídí, že request na /api/public/profiles se přesměruje na server.cityvizor.cesko.digital což je hostname serveru co žije ve stejném compose stacku jako landing-page.

Akorát by mě teda zajímalo, proč je to specifikovaný na jeden konkrétní endpoint a proč tam není rovnou /api.